invenio_administration/marshmallow_utils.py

```diff
@@ -2,12 +2,14 @@
 #
 # This file is part of Invenio.
 # Copyright (C) 2022 CERN.
+# Copyright (C) 2023 KTH Royal Institute of Technology
 #
 # Invenio is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Invenio administration marshmallow utils module."""
 
+from invenio_pages.services.schemas import DynamicSanitizedHTML
```

Comment:
If possible, I would avoid adding a dependency on `invenio-pages` to `invenio-administration`.
I think that we should change the code: the part `custom_mapping[field_type.inner.__class__]},` should be smarter and first check of the type, then check for the parent classes (to traverse up the inheritance). If not found, than fail.
In this way, it will go up to the parent class and find `SanitizedHTML`.

WDYT?